netbox/core/views.py

```diff
@@ -320,7 +321,7 @@ def get(self, request, queue_index, status):
         table = self.get_table(data, request, False)
 
         # If this is an HTMX request, return only the rendered table HTML
-        if request.htmx:
+        if render_partial(request):
```

Comment:
IMHO this function should be renamed, the name makes it sound like it is rendering something not just for a true/false test.  maybe is_htmx_request?  I realize this is already in the codebase, but wanted to point it out.

Reply:
maybe:

`should_render_partial()` ?

Reply:
Yeah, that's a good call-out. The logic is bit more nuanced than "is it ~~cake~~ HTMX?" because we're considering the target element as well, but we also want to keep the name concise.

Reply:
I went with `htmx_partial()`.